src/hotspot/share/runtime/globals.hpp

```diff
@@ -2100,10 +2103,11 @@ const intx ObjectAlignmentInBytes = 8;
       "-XX:CRaCRestoreFrom and continue initialization if restore is "      \
       "unavailable")                                                        \
                                                                             \
-  product(ccstr, CRaCIgnoredFileDescriptors, NULL, "Comma-separated list "  \
-      "of file descriptor numbers or paths. All file descriptors greater "  \
-      "than 2 (stdin, stdout and stderr are excluded automatically) not "   \
-      "in this list are closed when the VM is started.")                    \
+  product(ccstr, CRaCIgnoredFileDescriptors, NULL, MANAGEABLE,              \
```

Comment:
Having this is considered only on init, this also does not fit MANAGEABLE concept (there is no need to change that over management)